workloads/kube-burner/env.sh

```diff
@@ -69,3 +70,7 @@ export POD_READY_THRESHOLD=${POD_READY_THRESHOLD:-5000ms}
 # Alerting
 export PLATFORM_ALERTS=${PLATFORM_ALERTS:-false}
 
+# Output and Comparisons
+export COMPARISON_CONFIG=${COMPARISON_CONFIG:-""}
```

Comment:
may be nice to give an example of how you can give multiple configs here. I would think that almost always it would be good to have use the files you added under the touchstone_configs folder. 

https://github.com/mohit-sheth/airflow-kubernetes/blob/1cba9bcbaab5f5ed2425e5b5a2bf85e45667a4ae/dags/openshift_nightlies/config/benchmarks/medium-control-plane.json#L49

Reply:
should we also add a value or two as a default?

Reply:
My intention was to not do any post benchmark analysis by default.
Hence https://github.com/mohit-sheth/plow/blob/53a419383c35109d750580c8fe628703208a76d6/workloads/kube-burner/common.sh#L191